hbase-server/src/main/resources/hbase-webapps/master/hbck.jsp

```diff
@@ -37,9 +37,17 @@
 <%@ page import="org.apache.hadoop.hbase.util.Pair" %>
 <%@ page import="org.apache.hadoop.hbase.master.CatalogJanitor" %>
 <%@ page import="org.apache.hadoop.hbase.master.CatalogJanitor.Report" %>
+<%@ page import="org.apache.hadoop.hbase.util.Threads" %>
 <%
+  final String cacheParameterValue = request.getParameter("cache");
+  boolean cache = Boolean.valueOf(cacheParameterValue);
```

Comment:
nit: Since we need `boolean`, we can directly use: `Boolean.parseBoolean(cacheParameterValue)`